File: Problems/Codeforces/Colorblindness.cpp
#include <bits/stdc++.h>
using namespace std;

int T, N;
string A, B;

int main()
{
    cin >> T;
    while (T--)
    {
        cin >> N >> A >> B;
        bool ans = true;
        for (int i = 0; i < N; i++)
        {
            if ((A[i] == 'R' || B[i] == 'R') && A[i] != B[i]) ans = false;
        }
        if (ans) cout << "YES\n";
        else cout << "NO\n";
    }
}

File: Problems/Codeforces/Colorblindness.py

T = int(input())
for _ in range(T):

    N = int(input())
    A = input()
    B = input()

    ans = True
    for i in range(N):
        if (A[i] == 'R' or B[i] == 'R') and A[i] != B[i]:
            ans = False
    
    if ans:
        print("YES")
    else:
        print("NO")
